.github/workflows/ci.yml

```
@@ -23,3 +23,7 @@ jobs:
       - name: Run pre-commit checks
         run: |
           pre-commit run --all-files
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)